filebeat/module/nginx/access/test/test.log-expected.json

```diff
@@ -286,8 +289,12 @@
         "http.version": "1.1",
         "input.type": "log",
         "log.offset": 1398,
+        "nginx.access.remote_ip_list": [
+            "localhost",
+            "localhost"
```

Comment:
why is localhost repeated here?

Reply:
It appears twice in the example log line:
```
localhost, localhost - - [29/May/2017:19:02:48 +0000] "GET /test2 HTTP/1.1" 200 612 "-" "Mozilla/5.0 (Windows NT 6.1; rv:15.0) Gecko/20120716 Firefox/15.0a2" "-"
```
Not sure how this log was generated.